main.py

```diff
@@ -1,7 +1,49 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+
+    for gen in ["Western", "Action", "Dramma"]:
```

Comment:
The genre name 'Dramma' seems to be a typo. It is later updated to 'Drama', but it would be more consistent to use 'Drama' from the start.